### DISCUSSION TITLE
Build Error: error: unknown type name 'llama_init_result'

### DISCUSSION THREAD

**Discussion by AyushyaChitransh** [mudler/LocalAI#3188] [Q&A] [answered]
**LocalAI version:**
9cfc9ac66f9933e5f915b0ebbb06c2f613bffbcf

**Environment, CPU architecture, OS, and Version:**
Darwin 23.5.0 Darwin Kernel Version 23.5.0: Wed May  1 20:16:51 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T8103 arm64

**Describe the bug**
While building from source, getting this error:
backend/cpp/llama-avx2/llama.cpp/examples/grpc-server/grpc-server.cpp:461:9: error: unknown type name 'llama_init_result'

**To Reproduce**
make build

**Expected behavior**

**Logs**
```
LocalAI/backend/cpp/llama-avx2/llama.cpp/examples/grpc-server/grpc-server.cpp:461:9: error: unknown type name 'llama_init_result'
        llama_init_result llama_init = llama_init_from_gpt_params(params);
        ^
```

**Additional context**

**Comment by mudler** [comment, marked as answer]
you'd probably need to run a `make clean` if you are building from source

**Reply by AyushyaChitransh** [reply]
Thanks, this resolved the error.
